.github/workflows/ci.yml

```diff
@@ -75,6 +75,11 @@ jobs:
         with:
           target: wasm32-unknown-unknown
 
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.11.0"
+
```

Comment:
the issue with wasm compatibility with new rust version. https://github.com/rustwasm/wasm-bindgen/issues/4211#issuecomment-2423990971

might be worth to wait until the mentioned issue is resolved